packages/features/bookings/Booker/components/HavingTroubleFindingTime.tsx

```diff
@@ -25,7 +28,7 @@ export function HavingTroubleFindingTime(props: Props) {
   if (props.isScheduleLoading || !props.dayCount) return null;
 
   return (
-    <div className="bg-default border-subtle absolute bottom-[-120px] flex w-full min-w-0 items-center justify-between rounded-[32px] border p-3 text-sm leading-none shadow-sm">
+    <div className="bg-default border-subtle mt-6 flex w-1/2 min-w-0 items-center justify-between rounded-[32px] border p-3 text-sm leading-none shadow-sm lg:w-1/3">
```

Comment:
Not responsive
<img width="447" alt="Screenshot 2024-01-30 at 11 13 21 AM" src="https://github.com/calcom/cal.com/assets/53316345/f276fdf8-e156-4d51-a271-ffd891276d93">

Reply:
Was meant to be hidden here - will push a fix

Reply:
Addressed -> https://github.com/calcom/cal.com/pull/13457/commits/8d042dadf7722bfb572dbe34fd89602a27742303